src/collider/src/collider.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 The Manifold Authors.
+// Copyright 2023 The Manifold Authors.
```

Comment:
Sorry, not what I meant. Would you mind reverting that replace all commit? I just mean you created several files, and each of them had a copied data that was old - I commented on one, but there were ~3?

Reply:
Will do, I should have clarified first.